config/7.1.0/securebanking/ig/routes/routes-service/01-ob-rs-metadata.json

```diff
@@ -56,11 +57,11 @@
           "type": "ScriptableFilter",
           "config": {
             "type": "application/x-groovy",
-            "file" : "ProcessRs.groovy"
+            "file": "ProcessRs.groovy"
           }
         }
       ],
-      "handler": "ReverseProxyHandler"
+      "handler": "ForgeRockClientHandler"
```

Comment:
Why change this?
IG is reverse proxying the RS for this route

Reply:
Ah good catch!

Reply:
Fixed